src/utils/tests/perms.test.js

```diff
@@ -0,0 +1,54 @@
+import * as permUtils from '../perms';
+
+describe('perms', () => {
+  let mockMember = {
```

Comment:
wonder if we should create a separate mock util for some household stuff like member, messages, etc. (out of scope for this PR though)

Reply:
Probably should at some point :P
we should do it like this: https://jestjs.io/docs/en/manual-mocks